tests/Features/Skip.php

```diff
@@ -54,3 +54,11 @@
         return $this->shouldSkip;
     })
     ->toBeFalse();
+
+it('can skip on specific php version')
+    ->skipOnPhp(PHP_VERSION)
+    ->assertTrue(false);
+
+it('can skip on php versions depending on operator')
+    ->skipOnPhp('7.4.0', '>')
```

Comment:
This API needs to be `->skipOnPhp('^7.4.0')` or `->skipOnPhp('>=7.4.0')` or  `->skipOnPhp('>7.4.0')`. Can you adjust that?

Reply:
That wouldn't work with `version_compare`, so we'd either have to pull in a package (e.g. [`composer/semver`](https://github.com/composer/semver)) or write our own parser.

I think `composer/semver` would work nicely with:

```php
Composer\Semver\Semver::satisfiedBy(PHP_VERSION, $version)
```

Reply:
Yeah it's better, I have same idea as @owenvoke but I think it's better to use `composer/semver` than creating a parser for it on a test framework. I'm working on it...

Thanks everyone!